src/emitter.js

```diff
@@ -28,7 +29,7 @@ const defaults = {
   parent: function( /*context*/ ){
     return null;
   },
-  context: this
+  context: window
```

Comment:
Here I decided to use window module, because this in es modules is not allowed.

Reply:
`context: null` would be clearer as we also support a headless mode (Node.js).

Reply:
So context is not used since it's always null?

Reply:
The context is used in each emit with `Function.apply()`.  If it's `null`, it would be `window` in the browser and `global` in Node.